galette/templates/default/components/dynamic_fields.html.twig

```diff
@@ -169,7 +169,7 @@
 <script type="text/javascript">
     {% if not masschange %}
     var _addLnk = function(){
-        return $('<a class="ui small compact icon green button" href="#"><i class="plus icon" title="{{ _T("New occurrence") }}"></i> <span class="sr-only">{{ _T("New occurrence") }}"</span></a>');
```

Comment:
Isn't there any differenciation in Fomantic when we want to hide an element (like icons `i`) or when we want not to be hidden for screen readers?
Since I have no experience with screen readers, I do not know if that really changes something, but from a semantic point of view, "hidden" does not mean the same as "screen-reader only".

Reply:
> Isn't there any differenciation in Fomantic when we want to hide an element (like icons `i`) or when we want not to be hidden for screen readers?

No there isn't.

>Since I have no experience with screen readers, I do not know if that really changes something, but from a semantic point of view, "hidden" does not mean the same as "screen-reader only".

I do not really have experience with screen readers neither. But I know that the class name doesn't change anything.

Moreover, if such technique should effectively be used to handle screen readers only elements, setting `display:none` is not always the best thing to do. In most cases, it is recommended to move the elements off-screen. But not if it concerns elements navigable with the keyboard (links, forms, ...) or unless they are repositioned to be visible on focus for sighted users... Well... Accessibility is a complex topic !

I will set back the differenciation in the CSS using the proper rule. That will not make galette more accessible, but that way it will keep the original intent.

Reply:
OK, you can let as is oi was just asking :)